### PR TITLE
 Fix #3551 option_audio_language.xml into single xml file

### LINKED ISSUE #3551: Merge option_audio_language.xml into single xml file
Currently there are 3 versions of `option_audio_language.xml` file merge it into single xml file.

We can use https://text-compare.com/ to compare two versions of this file and for all the differences we can create variables in `dimens.xml` file and use it accordingly.

**Note**: In PR, make sure you add before and after screenshot of mobile-portrait, mobile-landscape, tablet-portrait and tablet-landscape for comparison and make sure that there is not difference between before and after UI.

### PR DESCRIPTION
**Pull request by shankarpriyank**
## Explanation
 - Fix #3551 
 - Merged 2 files of  option_audio_language.xml 

## Essential Checklist

- [x] The PR title and explanation each start with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have their rationale included in the PR explanation.
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary code changes from Android Studio ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#undo-unnecessary-changes)).
- [x] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
- [x] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).

## For UI-specific PRs only

If your PR includes UI-related changes, then:
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see [RTL guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines))
- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing

| Before  | After |
| --------- | -------- |
mobile landscape|mob land
mobile potrait|mob pot
tab landscape|tab land
tab potrait|tab pot

### CODE REVIEW

**Comment by shankarpriyank** [comment]
PTAL @bkaur-bkj @rt4914

**Comment by rt4914** [comment]
@vrajdesai78 Can you please review this PR and guide @shankarpriyank to update the screenshots correctly for comparison. Landscape screenshots are not available at all. Also for comparison the screenshots should be side by side and not top/bottom. Similar for his other ongoing PR.

**Comment by vrajdesai78** [comment]
@shankarpriyank can you please update the screenshots for landscape and arrange them properly. You can take reference from #4181 for arranging screenshots. For landscape mode, you may need to turn on auto-rotate in your emulator. Also, your explanation should start from Fix #3551.

File: app/src/main/res/layout-sw600dp/option_audio_language.xml [hunk 1]
```diff
@@ -20,9 +20,9 @@
     android:background="@{commonViewModel.selectedFragmentIndex == viewModel.itemIndex ? @drawable/general_item_background_border_cyan : @drawable/general_item_background_border}"
```
Comment (vrajdesai78) [inline app/src/main/res/layout-sw600dp/option_audio_language.xml]: You need to delete this file and add this extra code to the main file

`android:background="@{commonViewModel.selectedFragmentIndex == viewModel.itemIndex ? @drawable/general_item_background_border_cyan : @drawable/general_item_background_border}"`

**Review by vrajdesai78 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout-sw600dp/option_audio_language.xml (hunk 1 above)
Comment (shankarpriyank) [reply app/src/main/res/layout-sw600dp/option_audio_language.xml]: Thanks  @vrajdesai78 but I don't think the tablet file can also be merged, please take a look at our discussion in the pr #3552

Comment (vrajdesai78) [reply app/src/main/res/layout-sw600dp/option_audio_language.xml]: I think it won't be an issue if you delete this file and add code in the main file. So, I would suggest first try to run by removing this file and if it throws an error or UI is changed then you can keep this file.

**Review by vrajdesai78 — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout-sw600dp/option_audio_language.xml (hunk 1 above)
Comment (shankarpriyank) [reply app/src/main/res/layout-sw600dp/option_audio_language.xml]: Okay

**Comment by shankarpriyank** [comment]
I have made the changes as requested PTAL @vrajdesai78

**Review by vrajdesai78 — APPROVED** [review]

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**